docs/semgrep-code/findings.md

```diff
@@ -228,9 +228,9 @@ To **fix a finding**:
 
 1. Update, or refactor the code such that the Semgrep rule pattern no longer matches the code.
 
-#### Removing a rule
+#### Removing a rulesets
```

Comment:
```suggestion
#### Removing a ruleset
```

Reply:
@armchairlinguist maybe this is a matter of taste, I was just thinking that we should keep this in the plural, as we say: "Disabling rules" (Also in the plural.) Your suggestion might make it easier to search maybe?

Reply:
I can also change Disabling rules to Disabling rule... let me know @armchairlinguist. :-)

Reply:
"Removing rulesets" is fine but you can't mix the plural with the singular article :)

Reply:
Oh of course! :D I forgot it there... last minutes of work before past time... what a shame. :D @armchairlinguist